packages/xchain-bitcoin/src/client.ts

```diff
@@ -251,12 +247,16 @@ class Client implements BitcoinClient, XChainClient {
   /**
    * Get the BTC balance of a given address.
    *
-   * @param {Address} address By default, it will return the balance of the current wallet. (optional)
+   * @param {number} index By default, it will return the balance of the 0th address
```

Comment:
`Address` ìs a type alias of `string`, it can't be  a `number`

Reply:
good catch, i had originally coded getBalance(index) but subsequently re-refaactored it to take any arbitrary address. 

fixed